lang/ja.json

```diff
@@ -12,15 +12,15 @@
   "Non-Fullscreen": "フルスクリーン以外",
   "Mute": "ミュート",
   "Unmute": "ミュート解除",
-  "Playback Rate": "再生レート",
-  "Subtitles": "サブタイトル",
-  "subtitles off": "サブタイトル オフ",
-  "Captions": "キャプション",
-  "captions off": "キャプション オフ",
+  "Playback Rate": "再生速度",
+  "Subtitles": "吹き替え・字幕１",
+  "subtitles off": "吹き替え・字幕１ オフ",
+  "Captions": "表題・説明・字幕２",
```

Comment:
We don't actually want subtitles and captions to be numbered here. Subtitles are the general term outside of the United States. Captions generally refer to Subtitles for the Hard of Hearing, that is, subtitles that also include other auditory cues.

Reply:
Sorry for the late reply
Even the part I know
There is basically only one subtitle in Japan.
However, non-Japanese videos have multiple subtitles
In this case, multiple subtitles are language names.